### PR TITLE
Implements new case study qemu

### PR DESCRIPTION
**Pull request by vulder**

### CODE REVIEW

**Comment by vulder** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Complexity increasing per file
==============================
- varats/projects/c_projects/qemu.py  3
         

Clones added
============
- varats/projects/c_projects/qemu.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/VaRA/VaRA-Tool-Suite/pullRequest?prid=4625012&bid=15506196)

File: varats/projects/c_projects/qemu.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+from benchbuild.utils.compiler import cc, cxx
+from benchbuild.utils.run import run
+from benchbuild.project import Project
+from benchbuild.utils.cmd import make, cmake, git
+from benchbuild.utils.download import with_git
+
+from plumbum import local
+
+from varats.paper.paper_config import project_filter_generator
+from varats.utils.project_util import (get_all_revisions_between,
```
Comment (boehmseb) [inline varats/projects/c_projects/qemu.py]:
```suggestion
from varats.utils.project_util import wrap_paths_to_binaries
```

File: varats/projects/c_projects/qemu.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+                                       wrap_paths_to_binaries, block_revisions,
```
Comment (boehmseb) [inline varats/projects/c_projects/qemu.py]:
```suggestion
```

File: varats/projects/c_projects/qemu.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+                                       BlockedRevisionRange, BugAndFixPair,
```
Comment (boehmseb) [inline varats/projects/c_projects/qemu.py]:
```suggestion
```

File: varats/projects/c_projects/qemu.py [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+                                       BlockedRevision)
```
Comment (boehmseb) [inline varats/projects/c_projects/qemu.py]:
```suggestion
```

File: varats/projects/c_projects/qemu.py [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+Project file for qemu.
+"""
+import typing as tp
+from pathlib import Path
+
+from benchbuild.settings import CFG
+from benchbuild.utils.compiler import cc, cxx
+from benchbuild.utils.run import run
+from benchbuild.project import Project
+from benchbuild.utils.cmd import make, cmake, git
```
Comment (boehmseb) [inline varats/projects/c_projects/qemu.py]:
```suggestion
from benchbuild.utils.cmd import make, cmake
```

**Review by boehmseb — CHANGES_REQUESTED** [review]

**Comment by vulder** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- varats/projects/c_projects/qemu.py  3
         

Clones added
============
- varats/projects/c_projects/qemu.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/VaRA/VaRA-Tool-Suite/pullRequest?prid=4625012&bid=15506196)

**Review by boehmseb — APPROVED** [review]
lgtm

**Pull request merged**